docs/data-sources/csms_secrets.md

```diff
@@ -0,0 +1,80 @@
+---
+subcategory: "Data Encryption Workshop (DEW)"
+---
+
+# huaweicloud_csms_secrets
+
+Use this data source to get the list of CSMS secrets.
+
+## Example Usage
+
+```hcl
+variable "secret_name" {}
+
+data "huaweicloud_csms_secrets" "test" {
+  name = var.secret_name
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `region` - (Optional, String) Specifies the region in which to query the data source.
+  If omitted, the provider-level region will be used.
+
+* `name` - (Optional, String) Specifies the name of the secret.
+
+* `secret_id` - (Optional, String) Specifies the ID of the secret.
+
+* `status` - (Optional, String) Specifies the secret status. Valid values are **ENABLED**, **DISABLED**,
+  **PENDING_DELETE** and **FROZEN**.
+
+* `enterprise_project_id` - (Optional, String) Specifies the enterprise project id.
+
+* `event_name` - (Optional, String) Specifies the event name related to the secret.
+
+## Attribute Reference
+
+In addition to all arguments above, the following attributes are exported:
+
+* `id` - The data source ID.
+
+* `secrets` - Indicates the secrets list.
+  The [secrets](#CSMS_secrets) structure is documented below.
+
+<a name="CSMS_secrets"></a>
+The `secrets` block supports:
+
+* `id` - The secret ID.
+
+* `name` - Indicates the secret name.
+
+* `status` - Indicates the secret status.
+
+* `kms_key_id` - Indicates the ID of KMS key used to encrypt secret.
+
+* `description` - Indicates the secret description.
+
+* `created_at` - Indicates the time when the secret created, in UTC format.
+
+* `updated_at` - Indicates the time when the secret updated, in UTC format.
+
+* `scheduled_deleted_at` - Indicates the time when the secret is scheduled to be deleted, in UTC format.
+  When the secret is not in the deletion plan, the value of this field is **null**.
```

Comment:
You can delete this explanation because the default value of this field in the code logic is an empty string, not `null`.